### DISCUSSION TITLE
Gelatin coating needs to be fresh?

### DISCUSSION THREAD

**Discussion by StierliSa** [IBEXImagingCommunity/ibex_imaging_knowledge_base#79] [Q&A] [answered]
Hi everyone, 

I was just wondering whether it is possible to coat the slides a couple of days before? Hence could I coat them today and then  store them in the fridge and use them in a couple of days

Thanks a lot,

**Comment by anita-gola** [comment, marked as answer]
Hi StierliSa!
Great question!
 We always coat and then use the slides/chambers right away. We keep everything at RT (on the bench) because we noticed tissue lifting when we placed the prepared slides (with tissue) in the -20C freezer.
 
As alternative options, you could try:
1) Coat slides with chrome alum gelatin. Dry 1 hour at 60C. Leave slides (in a slide box/covered from dust) several days at RT. I wouldn’t do fridge.
2) Section tissues onto coated slides. Dry 1 hour at 37C and overnight at RT. Keep prepared slides at RT. Use within the week.

Good luck with all your future IBEX-ing!
